homeassistant/components/sensor/cert_expiry.py

```diff
@@ -34,11 +35,20 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up certificate expiry sensor."""
-    server_name = config.get(CONF_HOST)
-    server_port = config.get(CONF_PORT)
-    sensor_name = config.get(CONF_NAME)
+    def run_setup(event):
+        """Wait until Home Assistant is fully initialized before creating.
 
-    add_devices([SSLCertificate(sensor_name, server_name, server_port)], True)
+        Delay the setup until Home Assistant is fully initialized.
+        """
+        server_name = config.get(CONF_HOST)
+        server_port = config.get(CONF_PORT)
+        sensor_name = config.get(CONF_NAME)
+
+        add_devices([SSLCertificate(sensor_name, server_name, server_port)],
+                    True)
+
+    # Wait until start event is sent to load this component.
```

Comment:
This is a bit of a redundant comment because it just says what the code does – but the code already says that. If something looks weird, it's better to say _why_ this is needed.

So maybe something like "To allow checking of the HA certificate we must first be running"